server_manager/www/ui_components/outline-support-form.ts

```diff
@@ -132,9 +140,12 @@ export class OutlineSupportForm extends LitElement {
   }
 
   private handleTextInput(e: Event) {
-    const key: keyof FormValues = (e.target as TextField)
-      .name as keyof FormValues;
-    const value = (e.target as TextField).value;
+    const target = e.target as TextField;
+    const key = target.name as keyof FormValues;
+    if (key !== 'email' && key !== 'subject' && key !== 'description') {
+      throw new Error(`Cannot handle unknown text field ${key}`);
+    }
+    const value = target.value;
```

Comment:
It didn't because the interface was either string or boolean, but my solution was a bit too lazy.

With your suggestion to create a generic `handleInput` we don't need to check explicit keys. We still need to do a runtime check, but this is cleaner.